### DISCUSSION TITLE
Firmware version mismatch?

### DISCUSSION THREAD

**Discussion by ve6rah** [OpenEVSE/open_evse#20] [General]
Wifi version 4.2.2 suggests upgrading the EVSE firmware to 8.2.3, however 8.2.3 states that it requires WiFi version 4.2.3 (which has not yet been released)

Can we get some clarity on this?

**Comment by KipK** [comment]
8.2.3 is still prerelease. However don't flash it yet. I2C temperature sensors seems to not beeing detected

**Comment by KipK** [comment]
There was a typo also, it needs 4.2.2 , anyway there's is countermesure in the wifi fw that prevent the AUTO_AMPACITY bug , no urge to upgrade evse fw.

**Comment by ve6rah** [comment]
Have you considered labeling the 8.x.x versions as beta or pre-release or something?
Right now everything on the pages both for the wifi module, and the evse, show that the 7.x.x versions are outdated and people should be on the "latest" 8.x.x. if that's not the case, there really needs to be something to say so!

**Comment by KipK** [comment]
It is marked as pre release

**Comment by ve6rah** [comment]
Just to be clear. The current recommended version is 8.2.2 then? Or 7.1.3?
I've seen conflicting information

**Comment by KipK** [comment]
Latest stable is 8.2.2 yes

edit: 8.2.3 has been released btw.
